types/react-native-video/index.d.ts

```diff
@@ -141,7 +141,8 @@ export interface VideoProperties extends ViewProps {
     hideShutterView?: boolean;
     allowsExternalPlayback?: boolean;
     audioOnly?: boolean;
-
+    
+    onBuffer?(data: {isBuffering:boolean}): void;
     onLoadStart?(): void;
     onLoad?(data: OnLoadData): void;
     onBuffer?(): void;
```

Comment:
There's already an **onBuffer** prop. You should replace the previous one instead of adding a new one.

Also, could you create a type for the **data** parameter, like the **OnLoadData** for the **onLoad** prop?